### PR TITLE
Always filter roles by given project and userid (bsc#1124864)

### PR DESCRIPTION
**Pull request by dirkmueller**
https://bugzilla.suse.com/show_bug.cgi?id=1124864

### CODE REVIEW

File: keystone_json_assignment/json.py [hunk 1]
```diff
@@ -179,17 +179,18 @@ def list_role_assignments(self, role_id=None,
...
-                if projects:
-                    for project in projects:
-                        role_assignments.append({
-                            'role_id': self.role_id,
-                            'user_id': user_id,
-                            'project_id': project
-                        })
-            # If user is not in the map then this is probably not an LDAP user
-            # so just return the SQL grants
+                map_user_id = self.useridmap[user_id]
```
Comment (cmurphy) [inline keystone_json_assignment/json.py]: This should just be the user name, it's confusing to call it *_user_id

**Review by cmurphy — APPROVED** [review]
Ouch :grimacing: 

Verified the issue and the fix

File: keystone_json_assignment/json.py (hunk 1 above)
Comment (dirkmueller) [reply keystone_json_assignment/json.py]: renamed to "user", it was called that way in other places..

**Comment by cmurphy** [comment]
Looks good, my +1 still applies

**Review by bbobrov — APPROVED** [review]

**Pull request merged**